lib/paths-cache.js

```diff
@@ -415,7 +415,7 @@ export default class PathsCache extends EventEmitter {
 
     const cmd = [
       'find',
-      '-L',
+      atom.config.get('autocomplete-paths-ignore-circle-symlinks.findOptions'), // default -L
```

Comment:
This is the only interesting part of this PR. Everything else should be ignored.